index.bs

```diff
@@ -2758,44 +2793,69 @@ Appeal by Advisory Committee Representatives</h3>
 
 	An [=Advisory Committee representative=] initiates an [=appeal=] by sending a request to the [=Team=],
 	and should also share this request with the [=Advisory Committee=].
-	The request should say “I appeal this Decision”
+	The request <em class=rfc2119>should</em> say “I appeal this Decision”
 	and identify the decision,
-	and may also include their rationale for appealing the decision.
+	and <em class=rfc2119>may</em> also include their rationale for appealing the decision.
```

Comment:
```suggestion
	and <em class=rfc2119>may</em> also include their rationale for making the [=appeal=].
```

Reply:
I guess the suggested works too, but I don't seen what's wrong with the original (this line predates this PR by the way, I merely added an rfc2119 class on "may").

Reply:
I'll live with either wording, though I'd suggest an added cross-reference (which might need different markup, to handle `appeal` vs `appealing`)
```suggestion
	and <em class=rfc2119>may</em> also include their rationale for [=appealing=] the decision.
```